torch/__init__.py

```diff
@@ -892,6 +892,10 @@ def compiled_with_cxx11_abi():
 from torch._ops import ops
 from torch._classes import classes
 
+# decompositions_for_jvp on torch.ops
+# Registers decompositions for jvp to the jit registry
+from torch._decomp import decompositions_for_jvp
+
```

Comment:
Does this make it public API? E.g. `torch.decompositions_for_jvp`. If so then we've got a problem

Reply:
yes, nice catch - fixed